dev/AppLifecycle/FileActivatedEventArgs.h

```diff
@@ -44,11 +44,11 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
             }
         }
 
-        static IActivatedEventArgs Deserialize(winrt::Windows::Foundation::Uri const& uri)
+        static winrt::Windows::Foundation::IInspectable Deserialize(winrt::Windows::Foundation::Uri const& uri)
         {
             auto query = uri.QueryParsed();
-            std::wstring verb = query.GetFirstValueByName(L"Verb").c_str();
-            std::wstring file = query.GetFirstValueByName(L"File").c_str();
+            auto verb = std::wstring(query.GetFirstValueByName(L"Verb"));
+            auto file = std::wstring(query.GetFirstValueByName(L"File"));
```

Comment:
Here, we look for a case-sensitive key. But when parsing the extension contract, we used case-insensitive keys. Seems internally inconsistent. URL rules say that keys in query strings are case-sensitive. Why deviate from the standard?